README.md

```diff
@@ -209,13 +209,13 @@ Consider adding the `vim-go` plugin to your `.vimrc`
 If you're using [vim-plug](https://github.com/junegunn/vim-plug), that would
 look something like:
 
-```
+```vim
 Plug 'fatih/vim-go'
 ```
 
 If you're using Pathogen, that would look something like:
 
-```
+```text
```

Comment:
I think this should be `vim` as well?

Reply:
I was just _shooting from the hip_, so I just did [a check](https://github.com/github/linguist/blob/master/lib/linguist/languages.yml) and `vim` should work